openedx/core/djangoapps/content/learning_sequences/models.py

```diff
@@ -113,6 +113,10 @@ class LearningSequence(TimeStampedModel):
     )
     usage_key = UsageKeyField(max_length=255)
 
+    # A URL-safe Base64-encoding of a blake2b hash of the usage key.
+    # For aesthetic use (eg, as a path parameter, to shorten URLs).
+    usage_key_hash = models.CharField(max_length=255)
+
```

Comment:
This is the one and only model change. It allows us to implement the `get_learning_sequence_by_hash` function.